core/arch/arm/kernel/unwind_arm32.c

```diff
@@ -367,10 +367,14 @@ void print_stack(int level)
 {
 	struct unwind_state state;
 
-	memset(&state, 0, sizeof(state));
+	memset(state.registers, 0, sizeof(state.registers));
```

Comment:
maybe this is a stupid question: why zero-ing only the `registers` sub structure ?